### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by fabgrel10**
Accomplishments:

- [x] Create a collection that keeps a list of books.
- [x] Create a function to add a new book to the collection.
- [x] Create a function to add a new book to the collection.
- [x] Data is preserved in the browser's memory by using localStorage.

### CODE REVIEW

**Comment by omar-labana** [comment]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project where linters checks are not passing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  const bookStorage = (books) => {
+    localStorage.setItem('books', JSON.stringify(books));
+  };
+
+  const displayBooks = () => {
+    const booksLocalStorage = JSON.parse(localStorage.getItem('books'));
+    books = booksLocalStorage;
+    let html = '';
+    booksLocalStorage.forEach((book, index) => {
```
Comment (dasileker) [inline index.js:15]: - [ ] The code is clean but this line + line 48 is preventing from pushing a new book.

CPT2111231611-615x240

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @fabgrel10,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Álvaro and Fabricio, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**